apps/internal/tasks.py

```
@@ -0,0 +1,62 @@
+    )
+    if task_logs.count() > 0:
+        logger.info('Re-exporting stuck task_logs')
+        logger.info('%s stuck task_logs found', task_logs.count())
+        # workspace_ids = task_logs.values_list('workspace_id', flat=True).distinct()
+        expense_group_ids = task_logs.values_list('expense_group', flat=True)
+        ormqs = OrmQ.objects.all()
+        for orm in ormqs:
+            if 'chain' in orm.task and orm.task['chain']:
+                for chain in orm.task['chain']:
+                    if len(chain) > 1 and isinstance(chain[1], list) and isinstance(chain[1][0], ExpenseGroup):
+                        if chain[1][0].id in expense_group_ids:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)